src/components/Board/BoardItem.tsx

```diff
@@ -0,0 +1,274 @@
+    item,
+    users,
+  } = props;
+  return (
+    <Card
+      elevated
+      variables={({ colorScheme }: SiteVariablesPrepared) => ({
+        elevation: isDragging
+          ? colorScheme.elevations[8]
+          : colorScheme.elevations[4],
+        hoverElevation: colorScheme.elevations[8],
+        backgroundColor: colorScheme.default.background,
+      })}
+      styles={{
+        position: "relative",
+        zIndex: 1,
+        margin: `0 ${((20 - scrollbarWidth) / 16).toFixed(4)}rem .5rem 1.25rem`,
+        width: "auto",
+        height: "auto",
+      }}
+      accessibility={gridCellBehavior}
+      {...draggableProps}
+      {...dragHandleProps}
+      aria-label={`${t["board item"]}, ${getText(t.locale, item.title)}`}
+    >
+      <Box
+        styles={{
+          borderRadius: "2px",
+          overflow: "hidden",
+        }}
+      >
+        {item.preview && boardItemCardLayout.previewPosition === "top" && (
+          <BoardItemPreview preview={item.preview} />
+        )}
+        <Card.Body
+          {...(!item.preview || boardItemCardLayout.previewPosition !== "top"
+            ? { styles: { marginTop: "1.25rem" } }
+            : {})}
+        >
+          <Text weight="semibold">{getText(t.locale, item.title)}</Text>
+          {item.subtitle && (
+            <Text
+              size="small"
+              variables={({ colorScheme }: SiteVariablesPrepared) => ({
+                color: colorScheme.foreground1,
+              })}
+            >
+              {getText(t.locale, item.subtitle)}
+            </Text>
+          )}
+        </Card.Body>
+        {item.preview &&
+          boardItemCardLayout.previewPosition === "afterHeader" && (
+            <BoardItemPreview preview={item.preview} />
+          )}
+        {item.body && (
+          <Card.Body>
+            {Array.isArray(item.body) ? (
+              item.body.map((bodyItem, bi) => (
+                <BoardItemBody
+                  locale={t.locale}
+                  textObject={bodyItem}
+                  key={`BoardItem__${item.itemKey}__${bi}`}
+                />
+              ))
+            ) : (
+              <BoardItemBody
+                locale={t.locale}
+                textObject={item.body as TTextObject}
+              />
+            )}
+          </Card.Body>
+        )}
+        {(item.users || item.badges) && (
+          <Card.Footer>
+            <Flex>
+              <Box
+                styles={{
+                  flex: "1 0 auto",
+                  display: "flex",
+                }}
+              >
+                {item.users && (
+                  <BoardItemUsers
+                    locale={t.locale}
+                    associatedUserKeys={item.users}
+                    users={users}
+                  />
+                )}
+              </Box>
+              {item.badges && <BoardItemBadges t={t} badges={item.badges} />}
+            </Flex>
+          </Card.Footer>
+        )}
+        <b
```

Comment:
what is this `<b>` tag for?

Reply:
This is for adding a stacking margin to the bottom of this container so spacing is consistent even with varying content. I should add `aria-role=presentation` to clarify, though.